doc/radosgw/dynamicresharding.rst

```diff
@@ -185,27 +190,28 @@ Troubleshooting
 
 Clusters prior to Luminous 12.2.11 and Mimic 13.2.5 left behind stale bucket
 instance entries, which were not automatically cleaned up. This issue also affected
-LifeCycle policies, which were no longer applied to resharded buckets. Both of
+lifecycle policies, which were no longer applied to resharded buckets. Both of
 these issues could be worked around by running ``radosgw-admin`` commands.
 
-Stale instance management
+Stale Instance Management
 -------------------------
 
-List the stale instances in a cluster that are ready to be cleaned up.
+List the stale instances in a cluster that are ready to be cleaned up:
+
```

Comment:
Might there be stale instances that are not ready?

Also above suggest "can be remediated" instead of "could be worked around"

Reply:
I propose "that may be cleaned up".